website/src/auth.tsx

```diff
@@ -2,52 +2,71 @@ import { HubCallback } from "@aws-amplify/core"
 import Amplify, { Auth, Hub } from "aws-amplify"
 import React, { createContext, useContext, useState } from "react"
 
-const UserContext = createContext<CognitoUser | null>(null)
+import {
+  CognitoUser,
+} from 'amazon-cognito-identity-js'
```

Comment:
Right now, I think this is a transitive dependency from the amplify library. Could you run `yarn add amazon-cognito-identity-js`?

Reply:
A transitive dependency is when a package is installed through another package. So if we are `dailp` and our `package.json` says "we depend on `xxx`" and `xxx` depends on `yyy`, then `yyy` is a transitive dependency of `dailp`.